crates/api_crud/src/community/update.rs

```diff
@@ -35,6 +35,15 @@ pub async fn update_community(
   context: Data<LemmyContext>,
   local_user_view: LocalUserView,
 ) -> LemmyResult<Json<CommunityResponse>> {
+  // Verify its a mod (only mods can edit it)
+  check_community_mod_action(
+    &local_user_view.person,
+    data.community_id,
+    false,
+    &mut context.pool(),
+  )
+  .await?;
```

Comment:
So if one moderator uploads a community icon and then another mod replaces it, the file wont be deleted right?